apps/site/components/Downloads/ReleaseModal/index.tsx

```diff
@@ -31,7 +29,7 @@ const ReleaseModal: FC<ReleaseModalProps> = ({
   });
 
   return (
-    <Modal open={isOpen} onOpenChange={closeModal}>
+    <Modal onOpenChange={onOpenChange} {...props}>
```

Comment:
```suggestion
    <Modal {...props}>
```

Shouldn't this just be this?